floodviz/linked_data_utils.py

```diff
@@ -0,0 +1,150 @@
+from datetime import date
+
+
+class LinkedData:
+    """
+    Generates JSON-LD based on gages and the flood event.
+    """
+    def __init__(self):
+        self.ld = self._blank_thing("WebSite")
+        self.ld.update({
+            "name": "Active flood visualization",
+            "about": "Visualization of a flood event based on USGS streamgage readings",
+            "datePublished": str(date.today()),
+            "publisher": {
+                "@context": "http://schema.org",
+                "@type": "Organization",
+                "name": "U.S. Geological Survey",
+                "alternateName": "USGS"
+            },
+        })
+
+        self.gages = []
+        self.dates = {}
+        self.location = []
+
+    @staticmethod
+    def _blank_thing(typename):
```

Comment:
The base of all the schema.org ontologies is actually called 'thing', and I think this is easier to remember.